### PR TITLE
Replace grunt-svg-sprite for our own SVG compression script

### LINKED ISSUE #676: Remove grunt-svg-sprite
### Problem

`grunt-svg-sprite` adds too many dependencies, (including a phantom-js download!).

I get that it's useful to be able to automate the optimization of SVGs, but it would be nicer if instead they can just be saved pre-optimized in the repo, without `grunt-svg-sprite` and all of its dependencies. Plus we need to remove dependencies in general (#372).

### Solution

Remove `grunt-svg-sprite` and run `npm dedupe`, add all pre-optimized SVGs (optimizing using whatever third-party tool you'd like) instead of originals to the repo.

### PR DESCRIPTION
**Pull request by sandrina-p**
Closes #676

### CODE REVIEW

File: .Gruntfile.babel.js [hunk 1]
```diff
@@ -483,10 +437,6 @@ function svgLoader (options) {
       const svgName = path.basename(filePath, '.svg')
       const svgClass = `svg-${svgName}`
       const svg = source
-        // remove width and height attrs to be customized only by CSS
-        // because SVGO can't do it at the moment (bug): https://github.com/svg/svgo/issues/871
-        .replace(/width="[a-zA-Z0-9:]*"/, '')
-        .replace(/height="[a-zA-Z0-9:]*"/, '')
```
Comment (sandrina-p) [inline .Gruntfile.babel.js:489]: Note: As I explained at https://github.com/svg/svgo/issues/871#issuecomment-533801800, it seems this was a bug from svg-sprite, not SVGO itself. So we don't need to do it manually anymore.

**Review by taoeffect — CHANGES_REQUESTED** [review]
So, quick note about process here: if you decide to do a different solution to an issue, that needs to be discussed in the issue before a PR is submitted. Since this PR does not close #676 (the solution is different), I'm marking this as "request changes" for now, and we can discuss on the Monday call.

**Comment by sandrina-p** [comment]
@taoeffect removed SVGO as a dependency. Now SVGs are compressed manually using [SVGOMG](https://jakearchibald.github.io/svgomg/). I updated the SVG README to reflect that.

I would like just to mention I don't agree we do it manually when we have an automated way of doing it. However, I understand the goal of reducing dependencies from this project, so let's move on with this.

**Review by taoeffect — APPROVED** [review]
Excellent! 🙌

**Pull request merged**